src/app/search/components/patient_search/patient_search.directive.js

```diff
@@ -22,16 +22,20 @@
         function PatientSearchController($log, commonService) {
             var vm = this;
 
-            vm.assembledDob = assembledDob;
             vm.errorCount = errorCount;
             vm.searchForPatient = searchForPatient;
+            vm.assembledDob = assembledDob; 
```

Comment:
Prefer these alphabetized. Generally speaking, I tried to start with the style guide at https://github.com/johnpapa/angular-styleguide and work from there. Some of the files are better organized than others